docs/framework/unmanaged-api/debugging/debugging-interfaces.md

```diff
@@ -296,7 +296,10 @@ This section describes the unmanaged interfaces that handle the debugging of a p
  A logical extension of `ICorDebugProcess`.  
   
  [ICorDebugProcess3 Interface](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess3-interface.md)  
- Controls custom debugger notifications.  
+ Controls custom debugger notifications.
+
+ [ICorDebugProcess4 Interface](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess4-interface.md)
```

Comment:
```suggestion
 [ICorDebugProcess4 Interface](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess4-interface.md)\
```

You need either the slash at the end or the two white spaces to break the line. We're moving away from relying on the two spaces at the end

![image](https://user-images.githubusercontent.com/12971179/52514299-3b99d600-2bc5-11e9-8269-918712ba9376.png)